install_config/aggregate_logging.adoc

```diff
@@ -3847,3 +3850,232 @@ cluster. Edit your non-cluster logging service (for example, `logging-es`,
+
+*Cannot resolve kubernetes.default.svc.cluster.local*
+
+This internal alias for the master must be resolvable by the included
+DNS server on the master. Depending on your platform, you can run the `dig` command (for example, in a container) against the master to
+check whether this is the case:
+
+----
+$ dig kubernetes.default.svc.cluster.local @localhost
+[...]
+;; QUESTION SECTION:
+;kubernetes.default.svc.cluster.local. IN A
+
+;; ANSWER SECTION:
+kubernetes.default.svc.cluster.local. 30 IN A   172.30.0.1
+----
+
+Older versions of cluster logging did not automatically define this internal
+alias for the master. You might need to upgrade your cluster in order to
+use aggregated logging. If your cluster is up to date, there might be
+a problem with your pods reaching the SkyDNS resolver at the master
+or the pod could have been blocked from running. You must resolve this
+problem before deploying again.
+
+*Cannot connect to the master or services*
+
+If DNS resolution does not return at all or the address cannot be
+connected to from within a pod (such as the *fluentd* pod), this could be an
+indication of a system firewall/network problem. You need to debug this problem.
+
+=== Troubleshooting related to ElasticSearch
+
+The following troubleshooting issues apply to the ElasticSearch components of the EFK stack.
+
+*Elasticsearch deployments never succeed and rollback to previous version*
+
+This situation typically occurs itself on {product-title} with cluster logging deployed on AWS.  Describing the Elasticsearch pods typically reveal issues re-attaching the pods
+storage:
+
+----
+$ oc describe pod <elasticsearch-pod>
+----
+
+Consider patching each Elasticsearch deployment configuration to allow more time for AWS to make the storage available:
+
+----
+$ oc patch dc <elasticsearch-deployment-config> -p '{"spec":{"strategy":{"recreateParams": {"timeoutSeconds":1800}}}}'
+----
+
+*Searchguard index remains red*
+
+This is a known issue related to upgrading and moving to a single SearchGuard index per
+cluster instead of one index per deployment configuration.  The Elasticsearch Explain API is used to discover the reason and removing the index to node assignment is required:
+
+----
+$ oc -c elasticsearch exec ${pod} -- es_util --query=".searchguard/_settings" -XPUT -d "{\"index.routing.allocation.include._name\": \"\"}"
+----
+
+*Elasticsearch pods never become ready*
+
+This is known issue when the initialization and seeding process fails which
+can be from a red `.searchguard` index.
+
+----
+for p in $(oc get pods -l component=es -o jsonpath={.items[*].metadata.name}); do \
+  oc exec -c elasticsearch $ES_POD -- touch /opt/app-root/src/init_failures;  \
+done
+----
+
+=== Kibana
+
+The following troubleshooting issues apply to the Kibana components of the EFK stack.
+
+*Looping login on Kibana*
+
+If you launch the Kibana console and login successfully, you are incorrectly redirected back to Kibana,
```

Comment:
s/login/log in 
 Per IBM style guide, we keep it as 2 words when used as a verb, but as one word when used as an adjective or noun